rust/src/io/writer.rs

```diff
@@ -126,13 +127,19 @@ impl<'a> FileWriter<'a> {
     /// Write a [RecordBatch] to the open file.
```

Comment:
Could you update the doc to mention that all record batches will be treat as one batch on disk.